MyHashSet.java

```diff
@@ -0,0 +1,41 @@
+// Time Complexity :O(n)

```

Comment:
this is correct
define n as well 
here n is the length of the array list you're using